examples/bluetooth/esp_ble_mesh/ble_mesh_node/onoff_server/main/board.c

```diff
@@ -48,6 +84,9 @@ static void board_led_init(void)
         gpio_set_level(led_state[i].pin, LED_OFF);
```

Comment:
When the chip is C3 or S3, this code segment can not be executed.

Reply:
I revised function about board_led_operation().
LED on/off operation is same as ESP32.